bridge/test/NFTBridge_test.js

```diff
@@ -17,6 +17,9 @@ contract("Bridge NFT", async function(accounts) {
   const tokenName = "The Drops";
   const tokenSymbol = "drop";
   const tokenBaseURI = "ipfs:/";
+  const sideTokenSymbolPrefix = "e";
+  const nullAddress = "0x0000000000000000000000000000000000000000";

```

Comment:
you can import the null address from utils https://github.com/rsksmart/tokenbridge/blob/master/bridge/test/utils.js#L164